pyproject.toml

```diff
@@ -45,6 +46,7 @@ known-first-party = [
   "common",
   "data_refresh",
   "database",
+  "elasticsearch_cluster",
   "maintenance",
   "oauth2",
   "providers",
```

Comment:
Please also include `"test"` here. It's a first party in the API and the ingestion server. See #3700 for more info.

Reply:
Great catch!